client/js/shout.js

```diff
@@ -182,7 +188,7 @@ $(function() {
 		chan.click();
 	});
 
-	socket.on("msg", function(data) {
+	function renderChatMessage(data) {
```

Comment:
`renderChatMessage` builds a message and returns it, while `renderChannelMessage` actually renders something, i.e. builds it **and** shove it into the DOM.
I would call this `buildChatMessage` or equivalent `buildMessage` is probably enough, but it's up to you) to avoid the confusion.